HerPublicWebsite/Program.cs

```diff
@@ -61,6 +61,14 @@ public static void Main(string[] args)
                     "Send policy team update email",
                     rjs => rjs.SendPolicyTeamUpdate(),
                     "0 7 * * 1");
+            
+            app
+                .Services
+                .GetService<IRecurringJobManager>()

```

Comment:
We should refactor these job registrations so that we only fetch the job manager once:
```
var recurringJobManager = app.Services.GetService<IRecurringJobManager>();
```

While we are doing that, let's also make a couple of fixes to the refactored lines:

- Change `rjs =>` to `service =>` in each case
  - This is a relic of the removed `RecurringJobService`, so we should improve the naming
- Add an inline comment for each cron job to explain the timing
  - Parsing cron strings isn't immediately obvious, so may as well help future devs
  - There are inconsistent and incorrect comments at the minute, so let's remove those in favour of inline ones

So for each job, it should be something like:
```
recurringJobManager.AddOrUpdate<ReferralFollowUpNotificationService>(
    "Get referrals passed ten day working threshold with no follow up",
    service => service.SendReferralFollowUpNotifications(),
    "30 0 * * *"); // At 00:30 every day
```